### PR TITLE
Improved Python detection on Windows -- added registry path for MS Store installations

### PR DESCRIPTION
**Pull request by denis-gz**

### CODE REVIEW

File: osquery/tables/system/python_packages.cpp [hunk 1]
```diff
@@ -48,8 +48,10 @@ const std::set<std::string> kDarwinPythonPath = {
 };
 // clang-format on
 
-const std::string kWinPythonInstallKey =
-    "SOFTWARE\\Python\\PythonCore\\%\\InstallPath";
+const std::set<std::string> kWinPythonInstallKey = {
+    "SOFTWARE\\Python\\PythonCore\\%\\InstallPath",
+    "SOFTWARE\\Microsoft\\AppModel\\Lookaside\\user\\Software\\Python\\PythonCore\\%\\InstallPath"
```
Comment (iko1) [inline osquery/tables/system/python_packages.cpp:53]: I think 'user' it's your local username on your machine. I suggest replacing it with '%' in order to make it generic.

File: osquery/tables/system/python_packages.cpp [hunk 2]
```diff
@@ -174,13 +176,15 @@ QueryData genPythonPackagesImpl(QueryContext& context, Logger& logger) {
...
-    // Enumerate any system installed python packages
-    auto installPathKey = "HKEY_LOCAL_MACHINE\\" + kWinPythonInstallKey;
-    genWinPythonPackages(installPathKey, results, logger);
-
-    // Enumerate any user installed python packages
-    installPathKey = "HKEY_USERS\\%\\" + kWinPythonInstallKey;
-    genWinPythonPackages(installPathKey, results, logger);
+    for (const auto& key: kWinPythonInstallKey) {
+      // Enumerate any system installed python packages
+      auto installPathKey = "HKEY_LOCAL_MACHINE\\" + kWinPythonInstallKey;
```
Comment (iko1) [inline osquery/tables/system/python_packages.cpp:181]: Please make sure it compiles, the type of kWinPythonInstallKey isn't `std::string`

File: osquery/tables/system/python_packages.cpp (hunk 1 above)
Comment (denis-gz) [reply osquery/tables/system/python_packages.cpp:53]: No, it's not. I have no user named "user" on my PC. I have checked this on another machine and the path is the same.

File: osquery/tables/system/python_packages.cpp (hunk 2 above)
Comment (denis-gz) [reply osquery/tables/system/python_packages.cpp]: Fixed

**Comment by directionless** [comment]
@denis-gz Are you able to sign the CLA?

**Comment by denis-gz** [comment]
> @denis-gz Are you able to sign the CLA?

I already did, but after committing. Should I commit something again?

**Comment by directionless** [comment]
> > @denis-gz Are you able to sign the CLA?
> 
> I already did, but after committing. Should I commit something again?

These commits may be from a different email than is associated with your github account. Easiest fix is to rewrite the commits from the email address associated with your account.

**Comment by denis-gz** [comment]
Will open new PR, recommitting the changes.

**Comment by directionless** [comment]
Rewriting these commits is also okay. Whatever is easier for you.

**Pull request closed without merging**